### ISSUE TITLE
Instrumentation improvements

### ISSUE THREAD

**Issue by kaylareopelle** [rails/solid_queue#192]
The README states instrumentation for this gem is coming soon: 
> _Improvements to logging and instrumentation, a better CLI tool, a way to run within an existing process in "async" mode, and some way of specifying unique jobs are coming very soon._ 

Is there an issue related to instrumentation improvements I can follow?

**Comment by rosa** [comment]
@kaylareopelle yes! https://github.com/rails/solid_queue/issues/103

I plan to work on that one next, I've just been working on other stuff not related to Solid Queue this week 😅

**Comment by kaylareopelle** [comment]
Thank you! No problem, I know how that is! 😄 

It's nice to see the LogSubscriber! Many observability vendors like to use [ActiveSupport::Notifications](https://api.rubyonrails.org/classes/ActiveSupport/Notifications.html) to aid recording traces, and metrics. Has incorporating notifications instrumentation been discussed?

**Comment by rosa** [comment]
@kaylareopelle yes, definitely! `ActiveSupport::Notifications` are part of this plan. I hope to start work on this this week!
